### DISCUSSION TITLE
New to GitHub Enterprise - when to have multiple organizations?

### DISCUSSION THREAD

**Discussion by jmaixl** [community/community#72753] [Enterprise] [answered]
### Select Topic Area

Question

### Body

Hi all,
We are new to GitHub Enterprise and needed some guidance / opinions on how to structure our organizations.

**Problem: we have read the documentation about whether to have a single or multiple organizations, but we do not know which approach is best.**

We are an umbrella of companies that have been acquired.
Some of these are pretty independent, some have been taken over, some work together.
Some have a small codebase (20 repos), some are large (many hundreds of repos).

We see pros and cons to merge or to remain separate organizations. Being new to GitHub, we are taking the suggestion of "fewer is better" at face-value. But we are also somewhat a set of "portfolio" companies.

We just don't want to make a wrong choice as some stakeholders are concerned about a messy, single organization. I was hoping to get some insight on experiences with single/multiple organizations.

**Comment by oussamaef06** [comment, marked as answer]
Deciding whether to have a single or multiple organizations in GitHub Enterprise can be a significant decision, especially when you're dealing with a complex structure of acquired companies. There are pros and cons to both approaches, and the choice you make should align with your specific needs and objectives. Here are some considerations for both single and multiple organizations:

Single Organization:

Pros:

Centralized Management: Easier management of users, permissions, and settings.
Cross-Repository Collaboration: Facilitates collaboration and code sharing among teams in different acquired companies.
Consistency: Promotes a consistent development culture and practices across all entities.
Cons:

Complex Permissions: Managing permissions can be more complex due to the diverse nature of your acquired companies.
Information Overload: With many repositories, it might be overwhelming to navigate and find what you need.
Risk of Mixing Projects: It's possible for projects to become intermingled, making it challenging to distinguish which repository belongs to which company.
Multiple Organizations:

Pros:

Isolation: Provides clear separation between the entities, which can be useful when some companies should operate independently.
Simplified Permissions: Easier permission management within each organization since you're dealing with a more homogenous group of repositories and users.
Less Risk: Reduces the risk of accidental cross-company collaboration.
Cons:

Reduced Collaboration: It might be more challenging to facilitate collaboration and code sharing between companies, potentially leading to duplicated efforts.
Increased Administrative Overhead: You'll need to manage multiple organizations separately, which can be more time-consuming.
Considering your situation as an umbrella of companies with varying levels of independence, it may be beneficial to use a hybrid approach where you create separate organizations for the highly independent entities and a central organization for the ones that need to collaborate closely. This way, you strike a balance between isolation and collaboration.

Key steps to make an informed decision:

Identify Your Priorities: Determine the most critical factors for your organization, such as independence, collaboration, and ease of administration.

Engage Stakeholders: Involve stakeholders from each acquired company in the decision-making process to ensure their needs and concerns are considered.

Pilot Period: Consider a pilot period where you experiment with the chosen approach to ensure it works well in practice.

Documentation and Training: Whichever approach you choose, ensure clear documentation and provide training to your users to maximize their productivity and minimize confusion.

Regular Review: Periodically review your organization structure to adapt to changing needs and circumstances.

Remember that it's not a one-size-fits-all decision. The right structure for your organization will depend on your specific situation, and it might be beneficial to consult with GitHub Enterprise support or a professional consultant to help with the decision-making process and implementation.
